vesper-export.py

```diff
@@ -14,30 +14,49 @@
 
 def exportNotes(cursor, destDir, markdownify=False):
     print "Exporting to '%s':" % destDir
-    query = "SELECT uniqueId,text,archived,attachmentUniqueID,tags from notes;"
-    for record in cursor.execute(query):
+
+    rowsNotes = cursor.execute("SELECT uniqueId,text,archived,thumbnailID,creationDate,textModificationDate from notes;").fetchall()
+    rowsTags = cursor.execute("SELECT tagID,noteID from tagsNotesLookup;").fetchall()
+
+    for record in rowsNotes:
         id = record[0]
         text = record[1].encode("utf-8") if record[1] else None
         archived = bool(record[2])
         imageId = record[3].encode("utf-8") if record[3] else None
-        tags = record[4].encode("utf-8") if record[4] else None
+        dateCreated = record[4]
+        dateModified = record[5] if record[5] != None else dateCreated
+        tags = ""
+
+        tagCount = 0
+        for tagRecord in rowsTags:
```

Comment:
Can you parse the tags into a dictionary[noteId:[tag names]] above the rowNotes loop? Then here the logic would be to pull out the array of tag names from the dictionary and join them, rather than parsing all the tags for each and every note.